src/components/Lottie/interface.ts

```diff
@@ -34,7 +34,7 @@ export interface ReactLottieState {
 }
 
 export interface ReactLottieOwnProps extends ReactLottieState {
-  animationRef?: React.MutableRefObject<AnimationItem>;
+  animationRef?: React.MutableRefObject<AnimationItem | undefined | null> | React.RefObject<AnimationItem | undefined | null>;
```

Comment:
usage of `React.RefObject<AnimationItem | undefined | null>` leads to this error
![image](https://user-images.githubusercontent.com/30532154/95414877-90ac6300-0937-11eb-8451-579a107b3426.png)